infrastructure/mergegroup.def

```diff
@@ -1,22 +1,25 @@
 BootStrap: docker
-From: ubuntu:24.04
+From: python:3
 
 %labels
-  org.opencontainers.image.source = "https://github.com/SIESTA-eu/wp15"
+    org.opencontainers.image.source = "https://github.com/SIESTA-eu/wp15"
 
-%files
+%setup
+    mkdir -p /tmp/python_script
+    cp mergegroup.py /tmp/python_script/mergegroup.py
 
 %post
-  ################################################################################
-  # general updates to the base image
-  ################################################################################
-  apt update -y
-  echo 'tzdata tzdata/Areas select Europe' | debconf-set-selections
-  echo 'tzdata tzdata/Zones/Europe select Amsterdam' | debconf-set-selections
-  DEBIAN_FRONTEND=noninteractive apt install -y tzdata
-
+    ################################################################################
+    # general updates to the base image
+    ################################################################################
+    apt update -y && apt-get install -y python3-pip
+    echo 'tzdata tzdata/Areas select Europe' | debconf-set-selections
+    echo 'tzdata tzdata/Zones/Europe select Amsterdam' | debconf-set-selections
+    DEBIAN_FRONTEND=noninteractive apt install -y tzdata
+    pip install numpy 
+  
 %environment
-  export LC_ALL=C
+    export LC_ALL=C
 
 %runscript
-  echo Nothing to see here, please come back later...
+    python /tmp/python_script/mergegroup.py
```

Comment:
So this python script lives outside the container (/tmp???)? Please copy the script into the container (using `%files`) and execute it there. See:
https://apptainer.org/docs/user/main/definition_files.html#files
```
%files
    /file1
    /file1 /opt
```